positron/modules/atom_browser_web_contents.js

```diff
@@ -33,6 +45,10 @@ let WebContents_prototype = {
     this._browserWindow._domWindow.location = url;
   },
 
+  getTitle: function() {
+    return "/* stub */";
```

Comment:
> maybe just return the empty string or null, and console.warn("not implemented")?

I return the empty string in 0853030, but I didn't add a `console.warn` call, as I'd prefer to implement warnings for all stubs in #32.